en/_posts/2016-04-22-introduction-to-docker-with-mesos-marathon.md

```diff
@@ -8,43 +8,44 @@ lang: en
+Frameworks are used to launch **tasks** on Mesos slaves. Each framework is composed of two main components: the **scheduler** and the **executor**. The scheduler registers with the Mesos master, to be able to receive resources offers. The executor is a process that will be launched on Mesos slaves to run the **tasks**.
 
-If you want more information about the Apache Mesos architecture, please read the really good [the architecture documentation](http://mesos.apache.org/documentation/latest/architecture/).
+If you want more information about the Apache Mesos architecture, please read the really good [Mesos architecture documentation](http://mesos.apache.org/documentation/latest/architecture/).
 
 ##What is Marathon?
 
-Marathon is an Apache Mesos framework to run Docker containers. It provides a lot of features like framework high availibility through an active/passive system, constraints to ensure containers to be launched on different hosts or racks, application health checks, and many other things. You can manager your applications and containers using a web UI, allowing you to easily launch new containers, scale your applications, and more.
+[Marathon](https://mesosphere.github.io/marathon/) is an Apache Mesos framework to run Docker containers. It provides a lot of features like framework high availibility through an active/passive system, anti-affinity constraints to ensure containers are launched on different hosts or racks, application health checks, etc. You can manage your applications and containers using a web UI, allowing you to easily launch new containers, scale your applications, and more.
 
 #What do we provide?
 
-The Docker PaaS lab is based on all these technologies, adding some other cool stuff. Our lab is stack managed, it means that **we** deploy and manage your infrastructure.
+The Docker PaaS lab is based on all these software stacks, adding some other cool stuff, like redirecting your application Logs to our PaaS Logs, and your application metrics our PaaS Timeseries.. Our lab is stack managed, it means that **we** deploy and manage your infrastructure.
 
 ##Your resources are dedicated
 
-We are not providing a pool of slaves where everybody launches containers. You have dedicated slaves from OVH Cloud, and dedicated resources. Your containers are running on your slaves, and only yours, without noisy neighbours.
+We are not providing a pool of slaves where everybody launches containers. You have **dedicated** Public Cloud instances, and dedicated resources. Your containers are running on your instances, and only yours, without noisy neighbours.
+During the beta, you'll be limited to a single (and free) [VPS SSD 3](https://www.ovh.com/fr/vps/vps-ssd.xml). However, you'll be able to add more resources as we advance, not limited to the OVH cloud.
 
 ##We manage everything, but...
```

Comment:
Suggestion: As a reader, when reading "but.." when quickly scanning, I understand "we don't manage all, but we say so". Which is not true :)